firebase_read_block.py

```diff
@@ -0,0 +1,20 @@
+from .firebase_base import FirebaseBase
+
+
+class FirebaseRead(FirebaseBase):
+
+    def process_signals(self, signals):
+        out_sigs = []
+        for sig in signals:
+            # The collection may be different for different signals,
+            # so we need to compute it inside the loop
```

Comment:
This is definitely just my opinion, but this comment (and the one about enriching this signal) seem unnecessary. I don't like comments that just explain in english what the next line of code is doing, especially if that line of code isn't particularly confusing.